File: easy/testing.r
cat(sapply(readLines(tail(commandArgs(), n=1)), function(s) {
  t <- strsplit(s, " | ", fixed=TRUE)[[1]]
  r <- 0
  for (i in 1:nchar(t[1])) {
    if (substr(t[1], i, i) != substr(t[2], i, i)) {
      r <- r + 1
    }
  }
  if (r == 0) {
    "Done"
  } else if (r <= 2) {
    "Low"
  } else if (r <= 4) {
    "Medium"
  } else if (r <= 6) {
    "High"
  } else {
    "Critical"
  }
}), sep="\n")

File: easy/testing.lua
local m = {"Done", "Low", "Medium", "High", "Critical"}

function prio(a)
  if a == 0 then
    return 1
  elseif a <= 2 then
    return 2
  elseif a <= 4 then
    return 3
  elseif a <= 6 then
    return 4
  else
    return 5
  end
end

for line in io.lines(arg[1]) do
  local sep = line:find("|")
  local r = 0
  for i = 1, sep - 2 do
    if line:sub(i, i) ~= line:sub(sep+1+i, sep+1+i) then
      r = r + 1
    end
  end
  print(m[prio(r)])
end
